Makefile

```diff
@@ -44,6 +44,8 @@ clean:
 	mkdir -p build/locale
 
 locale:
+	# For generating a new file to let locales name localizable
+	python scripts/translate_locales.py
```

Comment:
I think this belongs in ``localeextract``, which is run periodically to generate new strings, and not ``locale``, which is used to add a new locale.

In either case, it fails in my VM with this traceback:

```
# For generating a new file to let locales name localizable
python scripts/translate_locales.py
Traceback (most recent call last):
  File "scripts/translate_locales.py", line 23, in <module>
    for lang_code, lang_info in settings.LOCALES.items()
  File "/home/vagrant/env/local/lib/python2.7/site-packages/django/conf/__init__.py", line 48, in __getattr__
    self._setup(name)
  File "/home/vagrant/env/local/lib/python2.7/site-packages/django/conf/__init__.py", line 44, in _setup
    self._wrapped = Settings(settings_module)
  File "/home/vagrant/env/local/lib/python2.7/site-packages/django/conf/__init__.py", line 92, in __init__
    mod = importlib.import_module(self.SETTINGS_MODULE)
  File "/usr/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
ImportError: No module named kuma.settings.testing
make: *** [localeextract] Error 1
```

This suggests it may be better as a Django management command, like ``manage.py extract`` and ``manage.py merge``, and automatically get the settings that way.